hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileTestUtils.java

```diff
@@ -58,8 +59,8 @@ private FileTestUtils() {
     static Key mockPayerLookup(Key key, AccountID accountId, ReadableAccountStore accountStore)
             throws PreCheckException {
         final var account = mock(Account.class);
-        given(accountStore.getAccountById(accountId)).willReturn(account);
-        given(account.key()).willReturn(key);
+        lenient().when(accountStore.getAccountById(accountId)).thenReturn(account);
```

Comment:
what is the reason for this change?

Reply:
the unit test that using this mockPayerLookup is using leginnt I just changed the syntax but the logic is the same
`        lenient().when(accountStore.getAccountById(accountId)).thenReturn(account);
        lenient().when(account.key()).thenReturn(key);`

Reply:
Ok